copilot/fastapi/app/kubernetes_api.py

```
@@ -0,0 +1,110 @@
+        HTTPException: If the Kubernetes API is unavailable or returns an error.
+    """
+    settings = get_settings()
+    
+    # Determine headers
+    try:
+        with open(settings.kubernetes_token_path, "r") as f:
+            kubernetes_token = f.read().strip()
+        headers = {
+            "Authorization": f"Bearer {kubernetes_token}",
+        }
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)